xla/stream_executor/cuda/cuda_blas_lt.cc

```diff
@@ -36,6 +36,9 @@ limitations under the License.
 #include "xla/stream_executor/scratch_allocator.h"
 #include "xla/stream_executor/stream.h"
 
+// TODO 
+// #if GOOGLE_CUDA ??
```

Comment:
What about this TODO? I think the #if GOOGLE_CUDA might not be needed, as this code is guarded with if_cuda_is_configured.

Reply:
yes, you are right, I was confused by ROCm side where we have an additional macro TF_HIPBLASLT which might be set to 0 even if ROCm is configured. I will remote this